tool/lib/commands/update_dart_sdk_deps.dart

```diff
@@ -76,7 +76,7 @@ class UpdateDartSdkDepsCommand extends Command {
         ),
         // TODO(kenz): is there a way to automatically close the file that pops up
         // with the commit description?
-        CliCommand('git cl upload -s'),
+        CliCommand('git cl upload -s -f'),
```

Comment:
what does -f do?

Reply:
` -f, --force           force yes to questions (don't prompt)`
It was the way i found to skip the prompts for a message.
I'll try the -m flag if that is less intense though :)

Reply:
so with the -f flag, does the file that pops up with the commit description no longer open? if so, can you remove the todo above this line?